src/views/Timeline.vue

```diff
@@ -40,7 +40,8 @@ export default {
     },
   },
   mounted: function () {
-    this.getBuckets();
+    (this.daterange = [moment().subtract(this.timeintervalDefaultDuration, 'seconds'), moment()]),
```

Comment:
@ErikBjare I think I might have just forgot to check this and I wasn't being careful enough with the rebase, my bad. My intention was to look out for the case where `timeintervalDefaultDuration` is null because `localStorage.durationDefault` is null. I now removed this `mounted` function in Timeline.vue and made similar changes in Bucket.vue. The initial request will be called by `valueChanged` in a `mounted` function in InputTimelineInterval.vue.